frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -871,9 +916,36 @@ export default function ClubEditCard({
     : creationDefaults
 
   return (
-    <Formik initialValues={initialValues} onSubmit={submit} enableReinitialize>
-      {({ dirty, isSubmitting }) => (
+    <Formik
```

Comment:
Is Formik used for anything besides overriding the onSubmit?

Reply:
The authors of the page definitely did not use many of FormiK's functionality (particularly with validation), but the page uses FormiK's isSubmitting and dirty attributes and most importantly it's submission feature (both initialValues and onSubmit). I just looked into FormiK more, and decided the best approach to add this modal will be taking advantage of its functionality more.